src/test/java/org/gridsuite/modification/server/service/ModificationRepositoryTest.java

```diff
@@ -449,23 +451,40 @@ public void testLineCreation() {
     }
 
     @Test
-    public void insertModificationTest() {
+    public void testMoveModificationInSameGroup() {
```

Comment:
That's quite a lot of DB statements :/ There is no way to by-pass this temporary table creation ? Maybe using JPA properties directly (previous comment) will do the trick, it could be nice to test this solution.

Reply:
I dont update order/group manually anymore. JPA manages everything, and there is no more tmp table, then no more insert/remove.